docs/development/howto/accessibility.md

```diff
@@ -0,0 +1,42 @@
+# Accessibility
```

Comment:
This is a useful new page. There are a bunch of grammatical and usage errors that are too difficult to make PR comments on. Maybe ping me after this gets merged, and I'll create a new PR to fix those problems. None are bad enough to prevent merging this, I think.